lumen/target.py

```diff
@@ -490,6 +490,9 @@ def _rerender(self, *events, invalidate_cache=False, update_views=True):
         if self._stale or (update_views and rerender):
             self._application._render()
             self._stale = False
+        # Remove the loading spinner set when a target needs to be
+        # rerendered, for instance when a view has not implemented _get_params.
+        self._application._layout.loading = False
```

Comment:
Only needs to happen if it actually had to re-render right, i.e. does it still work if you indent this line?

Reply:
Yep you're right it still works when this is executed in this `if` block (https://github.com/holoviz/lumen/pull/225/commits/985f2ca4f5136281656ea6a095db0f483c2837c2).